### PR TITLE
Adjust spec for KeywordValue to be wrapper for string, add table of properties to StyleValue types.

### PR DESCRIPTION
**Pull request by wilddamon**
Apparently I actually did this last year and completely forgot!

Keywords are now just a wrapper for a String.

I also started a table for which properties can take which StyleValues, but got stuck with how to make a table in bikeshed. Any advice?

### CODE REVIEW

**Comment by shans** [comment]
At a minimum, you can just use raw HTML. There might be a nicer way though - @tabatkins?

File: css-typed-om/Overview.bs [hunk 1]
```diff
@@ -221,6 +211,19 @@ The value for a given property is the last valid value provided in the string.
...
+
+<pre class='idl'>
+[Constructor(DOMString)]
+interface KeywordValue : StyleValue {
+  attribute DOMString keywordValue;
+};
+</pre>
+
+{{KeywordValue}} objects represent {{StyleValue}}s that are keywords. It is simply a wrapper for a string
+and does not check that it contains a valid keyword.
```
Comment (shans) [inline css-typed-om/Overview.bs]: Add a note that property setters will enforce keyword validity.

**Comment by tabatkins** [comment]
> but got stuck with how to make a table in bikeshed. Any advice?

Just write an HTML table. Bikeshed's source format is a subset of Markdown + HTML + some BS-specific stuff, so if it's not covered by vanilla Markdown, it's probably HTML.

Use good, semantic markup (thead/tbody, th, col/colgroup if appropriate) and then add `class=data` to get formatting automagically.  (fantasai put a lot of effort into the data-table styling and it shows.)

**Comment by wilddamon** [comment]
Added some updates and a big partial table for accepted types.

**Pull request merged**